### PR TITLE
Aggregates pages

### PR DESCRIPTION
**Pull request by amdprophet**
Depends on https://github.com/sensu/uchiwa/pull/255

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ angular.module('uchiwa').config(['$routeProvider', '$tooltipProvider',
       .when('/info', {templateUrl: 'bower_components/uchiwa-web/partials/views/info.html', controller: 'info'})
       .when('/stashes', {templateUrl: 'bower_components/uchiwa-web/partials/views/stashes.html', reloadOnSearch: false, controller: 'stashes'})
       .when('/settings', {templateUrl: 'bower_components/uchiwa-web/partials/views/settings.html', controller: 'settings'})
+      .when('/aggregates', {templateUrl: 'bower_components/uchiwa-web/partials/views/aggregates.html', reloadOnSearch: false, controller: 'aggregates'})
+      .when('/aggregates/:check', {templateUrl: 'bower_components/uchiwa-web/partials/views/check_aggregates.html', reloadOnSearch: false, controller: 'check_aggregates'})
+      .when('/aggregates/:check/:issued', {templateUrl: 'bower_components/uchiwa-web/partials/views/check_issue_aggregates.html', reloadOnSearch: false, controller: 'check_issue_aggregates'})
```
Comment (palourde) [inline js/app.js]: The templateUrl values are good however the HTML files do not exist, I still have these:

partials/aggregates/index.html
partials/check_aggregates/index.html
partials/check_issue_aggregates/index.html

File: partials/aggregates/index.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          <table class="table table-hover table-condensed">
+            <thead>
+              <tr>
+                <th ng-click="predicate = 'check'; reverse=!reverse">Check <i class="fa fa-sort"></i></th>
+                <th ng-click="predicate = 'issued'; reverse=!reverse">Last Issued <i class="fa fa-sort"></i></th>
+                <th ng-click="predicate = 'dc'; reverse=!reverse"><i class="fa fa-cloud" tooltip-placement="top" tooltip="Datacenter"></i> <i class="fa fa-sort"></i></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="aggregate in aggregates | filter:filters.q | filter:{dc:filters.dc} | limitTo:filters.limit | orderBy:predicate:reverse" ng-click="go('/aggregates/'+aggregate.check)">
```
Comment (palourde) [inline partials/aggregates/index.html]: `aggregates` variable isn't set anywhere. Probably need to add something like `$rootScope.aggregates = data.Aggregates;` in *backendService.update()*

File: partials/check_aggregates/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<div class="container-fluid client">
+  <div class="row-fluid">
+    <div class="col-lg-12">
+      <ol class="breadcrumb">
+        <li><a href="#/aggregates">Aggregates</a></li>
+        <li class="active">{{aggregates[0].check}}</li>
```
Comment (palourde) [inline partials/check_aggregates/index.html]: Looks like we are always showing the first element of the *aggregates* object no matter which aggregate is selected.

**Comment by amdprophet** [comment]
Thanks for catching these issues for me. It seems I lost some code when trying to bring my changes over to the new uchiwa-web repo. I'll push fixes for this tomorrow.

**Comment by amdprophet** [comment]
@palourde ready to be reviewed again

File: js/controllers.js [hunk 4]
```diff
@@ -389,6 +389,77 @@ controllerModule.controller('sidebar', ['$location', '$scope', 'userService',
...
+    $scope.pageHeaderText = 'Aggregates';
+    titleFactory.set($scope.pageHeaderText);
+
+    // Services
+    $scope.go = routingService.go;
+    $scope.permalink = routingService.permalink;
+
+    $scope.check = decodeURI($routeParams.check);
+    $scope.issued = decodeURI($routeParams.issued);
+    $scope.dc = decodeURI($routeParams.dc);
```
Comment (palourde) [inline js/controllers.js]: `$rootScope.dc` is already used in the navbar and contains an array of all datacenter so we should rename this local variable to something like `$scope.dcId`

File: partials/views/check_issue_aggregates.html [hunk 5]
```diff
@@ -0,0 +1,91 @@
+<ng-include src="partialsPath + '/navbar.html'"></ng-include>
+<ng-include src="partialsPath + '/sidebar.html'"></ng-include>
+<div class="container-fluid client">
+  <ng-include src="partialsPath + '/page-header.html'"></ng-include>
+  <div class="row-fluid">
+    <div class="col-lg-12">
+      <ol class="breadcrumb">
+        <li><a href="#/aggregates">Aggregates</a></li>
+        <li>
+          <a ng-href="#/aggregates/{{check | encodeURIComponent}}">
```
Comment (palourde) [inline partials/views/check_issue_aggregates.html]: This link won't contain the DC name so it show an empty page.

We could also use the format `/aggregates/dc/check` as used on the client view!

File: js/controllers.js (hunk 4 above)
Comment (amdprophet) [inline js/controllers.js]: Will do!

**Comment by amdprophet** [comment]
@palourde updated to match the style of client routes ^^

**Comment by palourde** [comment]
:+1:

**Pull request merged**